app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -105,7 +108,7 @@ Your contract must last at least one year and you must be working as:
 
 * a language teacher – any language taught in an English state school is eligible, except English 
 
-If you are employed to teach more than one subject, physics, general or combined science or a language or languages must make up at least 50% of your time in the classroom. The Department for Education will validate this with the school as part of the application process. Check with your school if you're not sure whether you are eligible. 
+If you are employed to teach more than one subject, physics, general or combined science or a language or languages must make up at least 50% of your time in the classroom. Your school can tell you how much of your time is allocated to teaching eligible subjects. The Department for Education will then validate this with the school as part of the application process. 
```

Comment:
@laurahermionetennant92 think there might be a word missing from this sentence as it didn't quite make sense to me and I had to read it a few times. 
If you are employed to teach more than one subject, physics, general or combined science or a language or languages must make up at least 50% of your time in the classroom. 

Do you mean if you teach more than one subject **then** physics, general or combines science, or a language or languages must make up at least 50% of your time in the classroom?

Reply:
then probs makes it clearer, will add!